backend/models/appModels/Tax.js

```diff
@@ -0,0 +1,27 @@
+const mongoose = require('mongoose');
+mongoose.Promise = global.Promise;
+
+const taxSchema = new mongoose.Schema({
+  taxName: {
+    type: String,
+    default: false,
+  },
+  taxValue: {
+    type: Number,
+    default: true,
+  },
+  isDefault: {
+    type: Boolean,
+    default: false,
+  },
+  updated: {
```

Comment:
remove  updated field

Reply:
Made the necessary changes you requested on the model